tests/integration/targets/elb_target/runme.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
 set -eux
-
+unset PYTHONPATH
 # Test graceful failure for older versions of botocore
-source virtualenv.sh
-pip install 'botocore<=1.7.1' boto3
-ansible-playbook -i ../../inventory -v playbooks/version_fail.yml "$@"
+python3 -mvenv --clear _venv
+_venv/bin/pip install 'botocore==1.7.0' 'boto3==1.4.7' boto3
+ansible-playbook -i ../../inventory -v playbooks/version_fail.yml -e "ansible_python_interpreter=$(pwd)/_venv/bin/python" "$@"
```

Comment:
With 2.0.0 we won't support botocore < 1.16.0 any more I guess we can drop this one

Reply:
Well this simplify the situation by a lot.